jobs/ingestor_syslog/spec

```diff
@@ -47,10 +52,6 @@ properties:
   logstash_ingestor.syslog_tls.ssl_key:
     description: Syslog-TLS SSL key (file contents, not a path) - required if logstash_ingestor.syslog_tls.port set
 
-  logstash_ingestor.relp.port:
-    description: Port to listen for RELP messages
-    default: 2514
-
```

Comment:
@jmcarp Overall it looks good, but could you add this part back, I think that they are lost during sync with upstream.